### PR TITLE
Updated check instructions with OSM IDs

### PR DESCRIPTION
**Pull request by maraarae**
### Description:

AtGradeSignPostCheck and EdgeCrossingEdgeCheck used Atlas IDs in their instructions.  The instructions have been updated to use OSM IDs instead of Atlas IDs.  DuplicateLocationInPolyLineCheck was also updated to include OSM IDs in check instructions.

### Potential Impact:

N/A

### Unit Test Approach:

Generated data and uploaded to MapRoulette to verify instructions.

### Test Results:

Successful.  Example image below shows that EdgeCrossingEdgeCheck now reports OSM IDs without comma-splitting in the instructions.
Screen Shot 2020-09-09 at 9 49 50 AM

### CODE REVIEW

**Comment by danielduhh** [comment]
Thanks for doing this @maraarae. Could you please add screenshots of the new instruction outputs in the PR comments "Test Results" section? Thanks!

File: src/main/java/org/openstreetmap/atlas/checks/validation/intersections/AtGradeSignPostCheck.java [hunk 1]
```diff
@@ -231,9 +231,9 @@ else if (roundAboutInEdgeToOutEdgeMap != null)
         final List<String> identifiers = this.getIdentifiers(entitiesToBeFlagged);
```
Comment (Bentleysb) [inline src/main/java/org/openstreetmap/atlas/checks/validation/intersections/AtGradeSignPostCheck.java:231]: This list also might need to use OSM IDs. That would need to be ubdated in `getIdentifiers`.

File: src/main/java/org/openstreetmap/atlas/checks/validation/intersections/EdgeCrossingEdgeCheck.java [hunk 2]
```diff
@@ -170,8 +170,9 @@ public boolean validCheckForObject(final AtlasObject object)
                 .filter(crossEdge -> crossEdge.getIdentifier() != edge.getIdentifier())
                 .flatMap(crossEdge -> getIntersection(edge, crossEdge).stream())
                 .collect(Collectors.toSet());
-        newFlag.addInstruction(this.getLocalizedInstruction(0, edge.getIdentifier(), collectedEdges
-                .stream().map(AtlasObject::getIdentifier).collect(Collectors.toList())));
+        newFlag.addInstruction(
+                this.getLocalizedInstruction(0, edge.getOsmIdentifier(), collectedEdges.stream()
+                        .map(AtlasObject::getIdentifier).collect(Collectors.toList())));
```
Comment (Bentleysb) [inline src/main/java/org/openstreetmap/atlas/checks/validation/intersections/EdgeCrossingEdgeCheck.java:175]:
```suggestion
                        .map(AtlasObject::getOsmIdentifier).collect(Collectors.toList())));
```

**Review by danielduhh — APPROVED** [review]
Thanks for testing!

**Review by dcastrowa — APPROVED** [review]
Nice job Mara! This update will add great value to challenges posted to MapRoulette.

**Review by Bentleysb — APPROVED** [review]

**Pull request merged**